pom.xml

```diff
@@ -20,7 +20,7 @@
         <spring.boot.test.version>2.6.7</spring.boot.test.version>
         <jackson.version>2.13.4</jackson.version>
         <lombok.version>1.18.24</lombok.version>
-        <hsqldb.version>2.4.1</hsqldb.version>
+        <hsqldb.version>2.7.1</hsqldb.version>
```

Comment:
Somewhere below there should be dependency using this version. You might need to add the classifier there as well.